content/events/2024-07-04-AskUsAnything/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Ask Us Anything about Galaxy webinar
+date: '2024-07-19'
+days: 1
+tease: "Th European Galaxy Team offers a Q&A session for all users"
+continent: EU
+location:
+  name: "online"
+contact: "Anika Erxleben-Eggenhofer"
+tags: [webinar]
+supporters:
+- eurosciencegateway
```

Comment:
```suggestion
- eurosciencegateway
- de.NBI
- de.KCD
```

Reply:
Is that ok?

Reply:
yes, we can add all who want to be mentioned here :)